junit-platform-console/junit-platform-console.gradle

```diff
@@ -61,6 +69,27 @@ artifacts {
 	archives shadowJar
 }
 
-distZip.dependsOn(shadowJar)
-distTar.dependsOn(shadowJar)
-installDist.dependsOn(shadowJar)
+task runStandaloneConsole(type: JavaExec) {
+	workingDir "$buildDir/libs"
+	main = '-jar'
+	args = [
+			"junit-console-$rootProject.version-standalone.jar",
+			'--scan-classpath',
+			'--details', 'none'
+	]
+	standardOutput = new ByteArrayOutputStream()
+	errorOutput = new ByteArrayOutputStream()
+}
+
+task checkStandaloneConsole {
+	doLast {
+		String text = runStandaloneConsole.errorOutput.toString() + runStandaloneConsole.standardOutput.toString()
+		println text
+		//assert text.contains("junit-jupiter (group ID: org.junit.jupiter, artifact ID: junit-jupiter-engine, version: $rootProject.version)")
+		//assert text.contains("junit-vintage (group ID: org.junit.vintage, artifact ID: junit-vintage-engine, version: $vintageVersion)")
```

Comment:
It's not clear to me if these asserts should be kept as-is, uncommented or deleted.